scalding-serialization/src/main/scala/com/twitter/scalding/serialization/macros/impl/OrderedBufferableProviderImpl.scala

```diff
@@ -16,7 +16,7 @@
 package com.twitter.scalding.serialization.macros.impl
 
 import scala.language.experimental.macros
-import scala.reflect.macros.Context
+import scala.reflect.macros.whitebox.Context
```

Comment:
we can possibly use https://github.com/milessabin/macro-compat if we want to escape any 2.11/2.12 warnings here, but I'd really like to keep 2.10 support as long as we can easily do so (even though twitter is actually on 2.11 these days).